CHANGELOG.md

```diff
@@ -83,6 +89,8 @@ fix: reference node iam arns inside the module for aws-auth configmap.
 
 [Unreleased]: https://github.com/boldlink/terraform-aws-eks/compare/3.0.2...HEAD
```

Comment:
This needs updating and a fun thing to also automate 💯 so add it to the backlog :D

Reply:
Alright